### PR TITLE
chore(chart): less labels on X axes and position Y axes on right

### PR DESCRIPTION
**Pull request by deanpress**
Improves the front-page price chart by displaying less labels on the X-axes, and positioning the Y-axes on the right side.

### CODE REVIEW

**Comment by ghost** [comment]
**Thanks for submitting this pull request!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

File: src/components/utils/TransactionAmount.vue [hunk 1]
```diff
@@ -70,7 +72,10 @@ export default class TransactionAmount extends Vue {
 
   get amountToSelf() {
     // @ts-ignore
-    if (this.transaction.sender === this.address && this.isMultiPayment(this.transaction.type, this.transaction.typeGroup)) {
+    if (
+      this.transaction.sender === this.address &&
+      this.isMultiPayment(this.transaction.type, this.transaction.typeGroup)
```
Comment (ItsANameToo) [inline src/components/utils/TransactionAmount.vue:78]: linter puts this on two lines, making the `@ts-ignore` fail to apply to the `isMultiPayment` call

Comment (ItsANameToo) [reply src/components/utils/TransactionAmount.vue:78]: should be able to resolve this with a `// prettier-ignore` line 🤔

Comment (ItsANameToo) [reply src/components/utils/TransactionAmount.vue:78]:
```
if (
      this.transaction.sender === this.address &&
      // @ts-ignore
      this.isMultiPayment(this.transaction.type, this.transaction.typeGroup)
) {
```

Comment (ItsANameToo) [inline src/components/utils/TransactionAmount.vue]:
```
if (
       this.transaction.sender === this.address &&
       // @ts-ignore
       this.isMultiPayment(this.transaction.type, this.transaction.typeGroup)
) {
```

Comment (ItsANameToo) [inline src/components/utils/TransactionAmount.vue:78]:
```suggestion
      // @ts-ignore
      this.isMultiPayment(this.transaction.type, this.transaction.typeGroup)
```

**Review by ItsANameToo — APPROVED** [review]

**Comment by ghost** [comment]
A member has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ghost** [comment]
Your pull request has been merged and marked as tier 4. _It will earn you $20 USD._

**Pull request merged**